### ISSUE TITLE
Remove Poor Docker Path Suggestions

### ISSUE THREAD

**Issue by bakerboy448** [linuxserver/docker-radarr#138]
[linuxserverurl]: https://linuxserver.io
[![linuxserver.io](https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver_medium.png)][linuxserverurl]

------------------------------

## Desired Behavior

Proper docker path suggestions [like here](https://wiki.servarr.com/Docker_Guide) and [here](https://trash-guides.info/Misc/how-to-set-up-hardlinks-and-atomic-moves/) in the suggested compose or no suggestions at all
## Current Behavior

Current behavior results in users having frequent issues due to improperly configured docker mounts. Also results in users using 2x the space for seeding torrents and extra wear and tear on the hard drives
## Alternatives Considered

Continue to frustrate users, application support staff, and LSIO support staff and confuse users as they will have a health check until this is resolved.

**Comment by aptalca** [comment]
Please stop opening countless issues. Numerous duplicate issues have been opened in the past on this.

**Comment by bakerboy448** [comment]
Then why not make the change(s) then.
Why continue to cause health checks in applications, intentionally cause excessive wear and tear on hard drives, and waste everyone’s time fixing the issues caused by the bad suggestions?

Correcting the suggested paths will save hundreds of hours of time in the world.

**Comment by aptalca** [comment]
Have you considered the possibility that we simply don't agree with you and have different goals and priorities than you?

**Comment by bakerboy448** [comment]
And what goals are those?

Assuming multiple mounts is part of those goals. Then I ask why do you have goals that are detrimental to the user and causes excessive wear and tear on their hardware do not seem like good goals to me.

Still waiting for those duplicate issues you claim to exist.

**Comment by aptalca** [comment]
The goal is to come up with a recommended sample config that works for most if not all users out of the box, and simply implemented.

What you're suggesting is to recommend a config that works really well for a small percentage of users who actually need it, but it's difficult to implement for most users and causes more confusion.

With that said, I'll be blunt. You're coming off as a little too aggressive and accusatory. If you don't tone it down, I'll bow out of the convo, fyi.

**Comment by bakerboy448** [comment]
We can switch over to discord for some more sanity/ease of comm; for posterity
![image](https://user-images.githubusercontent.com/55419169/111050776-3ea2c880-8414-11eb-8d49-7fb9ed0e622c.png)


>The goal is to come up with a recommended sample config that works for most if not all users out of the box, and simply implemented.

If use of the paths causes an error within the app and constant user support issues even if it does not. I would say that is neither working nor simple.

>What you're suggesting is to recommend a config that works really well for a small percentage of users who actually need it, but it's difficult to implement for most users and causes more confusion.

I'm suggesting to either make good suggestions or make no suggestions at all. Either one resolves the inherent issue at hand.

**Comment by bakerboy448** [comment]
@aptalca I think we got off on the wrong foot and I came in swinging the bat a bit too aggressively. For that, I apologize.

Some alt solutions briefly discussed on discord that would help overall without needing to redo the paths

- indicating the non-config paths are *optional* so users do not believe they are **required** as they frequently do
- adding some comments in the compose and docker run that tell users that ultimately only /config and one or more paths providing access to media and downloads is needed
- help text of some sort that shows in paths for unraid

I think these changes would significantly cut down on support on the app side, unraid side, and LSIO side.
Users commonly have the false believe that (in this case) the application requires /movies and /downloads when that is not the case.

Of course ideally users would actually read the docs, but that never happens.  One theory is users just use their docker GUI to find and pull the containers, never going near any documentation.
